modules/standard/Math.chpl

```diff
@@ -47,108 +41,191 @@ handling in the Math module.  The default behavior is as if the macro
 all math functions will return an implementation-defined value; no
 exception will be generated.
 
+Automatically Available Symbols
+-------------------------------
+
+.. note::
+
+   These symbols can also be accessed using ``Math.`` as their qualified access
+   prefix.
+
+.. _automath-roots:
+
+Roots
+^^^^^
+:proc:`~AutoMath.cbrt`
```

Comment:
I think it's unfortunate that the sections with links now have to be in a different file than where the functions are maintained.  I think that's going to make it a lot harder to remember to update the sections when new functions get added.

Reply:
I missed what all you and Brad tried, but would it be possible to `.. include::` the link sections from AutoMath instead of the whole file?

Reply:
This is definitely possible and actually looks much cleaner. Using the `:start-after:` and `:stop-before:` parameters, we can selectively import sections of AutoMath.

This actually removes the need for `@chpldoc.noheader`. Assuming the new approach looks good, I will remove the changes needed for this attribute from chpldoc.